### PR TITLE
App components

### PR DESCRIPTION
**Pull request by techEmprez**
The following took place:

1. Delete all the boilerplate from CRA (text, images, styles).
2. Create a directory called components.
3. Inside components, create a new Calculator.js file.
4. Style the UI

### CODE REVIEW

File: src/components/calculator.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
```
Comment (mateo951) [inline src/components/calculator.js:1]: `Calculator.js` file inside the components directory ✔

File: src/components/calculator.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+import './calculator.css';
+
+// eslint-disable-next-line react/prefer-stateless-function
+class Calculator extends React.Component {
```
Comment (mateo951) [inline src/components/calculator.js:5]: `Calculator.js` component is class-based 😎

File: src/App.js [hunk 3]
```diff
@@ -1,29 +1,18 @@
...
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+// eslint-disable-next-line react/prefer-stateless-function
+class App extends React.Component {
```
Comment (mateo951) [inline src/App.js:6]: The `App.js` component is class-based 🐐

**Review by mateo951 — APPROVED** [review]
### Hi @techEmprez, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 😁 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 3 above)
Comment (techEmprez) [reply src/App.js:6]: done

File: src/components/calculator.js (hunk 2 above)
Comment (techEmprez) [reply src/components/calculator.js:5]: done

Comment (mateo951) [reply src/components/calculator.js:5]: Just pointing out the key requirements you did well 😉

**Pull request merged**